src/napari/layers/shapes/shapes.py

```diff
@@ -2460,8 +2463,18 @@ def interaction_box(self, index):
             elif len(index) == 1:
                 box = copy(self._data_view.shapes[next(iter(index))]._box)
             else:
-                indices = np.isin(self._data_view.displayed_index, list(index))
-                box = create_box(self._data_view.displayed_vertices[indices])
+                disp_indices = [
```

Comment:
Consider renaming to `displayed_shape_indices`

```suggestion
                displayed_shape_indices = [
```